onnxruntime/core/so/cxa_demangle.cc

```diff
@@ -0,0 +1,39 @@
+#if USE_DUMMY_EXA_DEMANGLE
```

Comment:
If we're only using this for android, onnxruntime/platform/android would be a better location rather than creating a new directory with an unclear name (not sure what 'so' is meant to stand for).

Reply:
Moved